tasks/simplify.py

```diff
@@ -0,0 +1,55 @@
+from luigi import Task, Parameter
+import json
+import os
+from util import get_logger
+from simplification import SimplifyGeometriesMapshaper, SimplifyGeometriesPostGIS
+from meta import CurrentSession
+
+TABLE_ID = 'tableid'
+SIMPLIFICATION = 'simplification'
+GEOM_FIELD = 'geomfield'
+FACTOR = 'factor'
+MAX_MEMORY = 'maxmemory'
+SKIP_FAILURES = 'skipfailures'
+
+OBSERVATORY_SCHEMA = 'observatory'
+SIMPLIFICATION_MAPSHAPER = 'mapshaper'
+SIMPLIFICATION_POSTGIS = 'postgis'
+
+LOGGER = get_logger(__name__)
+
+
+def find_table_name(table_id):
+    session = CurrentSession().get()
+    return session.execute("SELECT tablename FROM observatory.obs_table WHERE id = '{tableid}'".format(
+                            tableid=table_id)).fetchone()[0]
+
+
+class SimplifyOne(Task):
+    table = Parameter()
+
+    def run(self):
```

Comment:
What about making this a wrapper task? I think it fits better, because this is not actually a dynamic dependency, the dependency can be known before runtime (it just depends on the static json and a parameter).

It would be a matter of:
- Inheriting from `WrapperTask`
- Putting the code in `requires` rather than `run`
- `return`ing instrad of `yield`ind the tasks.

Reply:
Makes a lot of sense as [the documentation](http://luigi.readthedocs.io/en/stable/luigi_patterns.html#triggering-many-tasks) states that:
> You’ll need to use `WrapperTask` for this instead of the usual `Task` class, because this job will not produce any output of its own, and as such needs a way to indicate when it’s complete. This class is used for tasks that only wrap other tasks and that by definition are done if all their requirements exist.